src/VirtoCommerce.CatalogModule.Core/Extensions/EnumerableExtensions.cs

```diff
@@ -0,0 +1,29 @@
+using System;
```

Comment:
The same method was added into the platform. Maybe need to remove it from here.

Reply:
Yes we can. But we need to raise the platform dependency to an unreleased version.

Reply:
Or release the platform first, and then use it here.